scenarioo-client/app/manage/generalSettings/generalSettings.html

```diff
@@ -54,12 +54,27 @@
             </div>
         </div>
 
+        <h2>Diff viewer</h2>
+
+        <div id="sc-diff-viewer-not-configured-message" class="row sc-space-bottom" ng-show="!vm.diffViewerAvailable" ng-cloak>
+            <div class="col-lg-12">
+                Image comparison is <span class="sc-service-disabled">disabled</span>. Structural comparisons are enabled.<br/>
+                See <a href="https://github.com/scenarioo/scenarioo/blob/{{vm.version.releaseBranch}}/docs/features/diff-viewer/setup.md" target="_blank">Setup instructions for Diff Viewer</a> on how to configure and setup Diff Viewer for Scenarioo.
+            </div>
+        </div>
+
+        <div class="row sc-space-bottom" ng-show="vm.diffViewerAvailable" ng-cloak>
+            <div class="col-lg-12">
+                Image comparison is <span class="sc-service-enabled">enabled</span>. See <a href="https://github.com/scenarioo/scenarioo/blob/{{vm.version.releaseBranch}}/docs/features/diff-viewer/setup.md#2-add-a-comparison-configuration" target="_blank">Comparison Configuration documentation</a> for more information about configuring Diff Viewer.
+            </div>
+        </div>
+
         <h2>Full Text Search</h2>
 
         <div id="sc-search-not-configured-message" class="row sc-space-bottom" ng-show="vm.searchEngineStatus && !vm.searchEngineStatus.endpointConfigured" ng-cloak>
             <div class="col-lg-12">
-                Full text search is disabled. <br/>
-                See <a href="https://github.com/scenarioo/scenarioo/blob/{{vm.version.releaseBranch}}/docs/features/full-text-search/setup.md" target="_blank">Setup Instructions for Full Text Search</a> on how to configure and setup Full text search for Scenarioo.
+                Full text search is <span class="sc-service-disabled">disabled</span>. <br/>
```

Comment:
There's no message for enabled, just input fields. That's why I couldn't format it the same.